src/main/java/org/reaktivity/nukleus/http_cache/internal/stream/HttpCacheProxyCacheableRequest.java

```diff
@@ -129,23 +132,27 @@ MessageConsumer newResponse(
         boolean retry = HttpHeadersUtil.retry(responseHeaders);
         MessageConsumer newStream;
 
-        if ((retry  &&
+        if (!isRequestPurged &&
+            ((retry  &&
```

Comment:
please move `(retry && attempts < 3)` onto one line to ease readability.

Reply:
Done.